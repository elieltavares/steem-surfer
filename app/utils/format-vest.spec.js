/* eslint-disable */
import formatVest from './format-vest';

describe('formatVest', () => {
  it('format number for vests', () => {
    let input = 58843.0;
    expect(formatVest(input)).toMatchSnapshot();
  });

  it('format number for vests 2', () => {
    let input = 558843.0;
    expect(formatVest(input)).toMatchSnapshot();
  });

  it('format number for vests 3', () => {
    let input = 9999999.0;
    expect(formatVest(input)).toMatchSnapshot();
  });

  it('format number for vests 4', () => {
    let input = 9999999.213213;
    expect(formatVest(input)).toMatchSnapshot();
  });
});
