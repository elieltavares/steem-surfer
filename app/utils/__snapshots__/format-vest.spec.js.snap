// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatVest format number for vests 1`] = `"58.843"`;

exports[`formatVest format number for vests 2 1`] = `"558.843"`;

exports[`formatVest format number for vests 3 1`] = `"9.999.999"`;

exports[`formatVest format number for vests 4 1`] = `"9.999.999,213"`;
